query-engine/connectors/sql-query-connector/src/database/operations/read.rs

```diff
@@ -152,7 +152,7 @@ pub async fn aggregate(
     group_by: Vec<ScalarFieldRef>,
     query_arguments: QueryArguments,
 ) -> crate::Result<Vec<AggregationRow>> {
-    if group_by.len() > 0 {
+    if group_by.is_empty() {
```

Comment:
This looks wrong. This misses a `!`, right?

Reply:
Oops, good catch!